fedimint-client-module/src/module/mod.rs

```diff
@@ -111,6 +111,10 @@ pub trait ClientContextIface: MaybeSend + MaybeSync {
         persist: EventPersistence,
     );
 
+    fn subscribe_persisted_events(
```

Comment:
I'm worried about having transient subscribers to persistent events. The whole point was for the downstream not to miss any events. They should subscribe, handle what they care about and _track their position_ themselves.